### ISSUE TITLE
Scrolling locked on mobile devices

### ISSUE THREAD

**Issue by marcreichel** [NightCatSama/vue-slider-component#413]
#### Describe the bug

The Slider seems to disable/break scrolling on mobile/touch devices. On desktop computers/non-touch devices scrolling works just fine.

#### Environment

 - OS & Version: iOS 13.2 (But also reproducible using the Chrome Device Toolbar)
 - Vue version: v2.5.17
 - Component Version: v3.0.41

**Comment by NightCatSama** [comment]
You can try adding a style (`touch-action: auto`) to the component.
```
<vue-slider v-model="value" :style="{ touchAction: 'auto' }" />
```

**Comment by marcreichel** [comment]
This worked, thank you. But shouldn’t this be the default behavior?

**Comment by NightCatSama** [comment]
It has been fixed in version 3.0.42.

Thanks.
